app/application/dto/account_dto.py

```
@@ -1,16 +1,14 @@
-class BalanceDTO:
+class BalanceDTO(BaseModel):
     currency: str
     balance: str
     locked: str
     avg_buy_price: str
     unit: str
 
 
-@dataclass
-class AccountBalanceDTO:
+class AccountBalanceDTO(BaseModel):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)